gobblin-metrics-libs/gobblin-metrics-base/src/main/avro/GaaSObservabilityEventV0.avsc

```diff
@@ -0,0 +1,135 @@
+{
+  "type": "record",
+  "name": "GaaSObservabilityEventExperimental",
+  "namespace" : "org.apache.gobblin.metrics",
+  "doc": "An experimental feature for GaaS to emit events during and after a job is executed.",
+  "fields": [
+    {
+      "name": "timestamp",
+      "type": "long",
+      "doc": "Time at which event was created in millis"
+    }, {
+      "name" : "flowGroup",
+      "type" : "string",
+      "doc" : "Flow group for the GaaS flow",
+      "compliance" : "NONE"
+    }, {
+      "name" : "flowName",
+      "type" : "string",
+      "doc" : "Flow name for the GaaS flow",
+      "compliance" : "NONE"
+    }, {
+      "name" : "flowExecutionId",
+      "type" : "long",
+      "doc" : "Flow execution id for the GaaS flow",
+      "compliance" : "NONE"
+    }, {
+      "name": "jobOrchestratedTime",
+      "type": "long",
+      "doc": "Timestamp when the job was successfully sent to the job executor, -1 if it was unable to be sent."
+    }, {
+      "name": "lastFlowModificationTime",
+      "type": "long",
+      "doc": "Timestamp in millis when the flow config was last modified"
+    }, {
+      "name" : "flowGraphEdgeId",
+      "type" : "string",
+      "doc" : "Flow edge id, in format <src_node>_<dest_node>_<edge_id>",
+      "compliance" : "NONE"
+    }, {
+      "name": "jobName",
+      "type": "string",
+      "doc": "The name of the Gobblin job, found in the job template. One edge can contain multiple jobs",
+      "compliance" : "NONE"
+    }, {
+      "name": "jobStatus",
+      "type": {
+        "type": "enum",
+        "name": "JobStatus",
+        "symbols": [
+          "SUCCEEDED",
+          "FAILED",
+          "CANCELLED"
+        ],
+        "doc": "Final job status for this job in the GaaS flow",
+        "compliance": "NONE"
+      }
+    }, {
+      "name": "jobStartTime",
+      "type": "long",
+      "doc": "Start time of the job in millis",
+      "compliance": "NONE"
+    }, {
+      "name": "jobEndTime",
+      "type": "long",
+      "doc": "Finish time of the job in millis",
+      "compliance": "NONE"
+    }, {
+      "name": "proxyUser",
+      "type": "string",
+      "doc": "Proxy user (if applicable) that ran the Gobblin job",
```

Comment:
what does "(if applicable)" mean?  I'd suggest always setting this to the effective user executing the job, so non-optional, rather than exploring making it nullable...

Reply:
What about Fliptop GaaS where there's no proxy user?

Reply:
what of the proposal to generalize this to `executionUserUrn`?